ycms/cms/views/ward/ward_management_view.py

```diff
@@ -0,0 +1,94 @@
+from django.contrib import messages
+from django.shortcuts import redirect, render
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...constants import job_types
+from ...forms import WardForm
+from ...models import User, Ward
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+
+    template_name = "ward/ward_management.html"
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        This function returns a list of all wards,
+        as well as a form for creating a new one
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+
```

Comment:
```suggestion

        :return: List of all wards
        :rtype: ~django.template.response.TemplateResponse
```
Not sure where that went! 😄